pkg/uefi/edk2-patches/edk2-stable202408/0000-Add-Xen-spoofing-in-XenPlatformPei.patch

```diff
@@ -0,0 +1,27 @@
+From 40f36e7c4c78caf19a8367995f615139190dfbd2 Mon Sep 17 00:00:00 2001
```

Comment:
Do we know why we need this patch?
I had the impression that it's somehow related to following GOP-related patches...

Reply:
It also could be put into commit message =)

Reply:
It is for Xen spoofing (not detectable) allowing Nvidia graphics passthrough without B.S.O.D'ing on Windows, I'll add some info in the commit message.

Reply:
added.